src/vaadin-custom-field.html

```diff
@@ -197,7 +185,8 @@
          */
         checkValidity() {
           if (this.required) {
-            return this.inputs.every(input => (input.validate || input.checkValidity).call(input));
+            return this.inputs.every(input => input.value) &&
```

Comment:
BTW is there a need to call `every` 2 times? One iteration should be enough.
Not a blocker.

Reply:
See the [description](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) for `.every`. It is _not_ an array iterator but a function that checks if all the items in an array pass the given condition. It's also not guaranteed that all the array items are iterated with `.every` because once an item that doesn't satisfy the condition is reached the function can skip checking/iterating the rest of the items and return `false`.

`.forEach` is the proper function for this purpose